.travis.yml

```diff
@@ -0,0 +1,50 @@
+sudo: required
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+    - apt-transport-https
+
+install:
+  - git fetch --unshallow --tags
+  - pip install PyYAML
+  - pip install virtualenv
+  - pip install reno
+  - |
+    test -e Gemfile || cat <<EOF > Gemfile
+    source 'https://rubygems.org'
+    gem 'rake'
+    gem 'test-kitchen'
+    gem 'kitchen-docker'
+    gem 'kitchen-inspec'
+    gem 'inspec'
+    gem 'kitchen-salt'   #, :git => 'https://github.com/salt-formulas/kitchen-salt.git'
+  - bundle install
+
+env:
+    - PLATFORM=trevorj/salty-whales:xenial-2017.7 SUITE=server-single
+    - PLATFORM=trevorj/salty-whales:trusty-2017.7 SUITE=server-single
+    - PLATFORM=trevorj/salty-whales:xenial SUITE=server-single
+    - PLATFORM=trevorj/salty-whales:trusty SUITE=server-single
```

Comment:
try to add PLATFORM=centos-7 SUITE=server-single
also you should have here all suites listed OR only the most complex ones.

Reply:
Done...

Reply:
I think that official centos7 image requires 'docker login'. I had good results with 'corux/centos7-salt/' image. 
@epcim Wouldn't it be better? Do you have it tested with official 'centos7' image?

Reply:
I skipped the centos tests...